test/infrastructure/net-operator/controllers/networkinterface_controller.go

```diff
@@ -106,10 +106,12 @@ func (r *NetworkInterfaceReconciler) Reconcile(ctx context.Context, req ctrl.Req
 
 // SetupWithManager will add watches for this controller.
 func (r *NetworkInterfaceReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, options controller.Options) error {
+	predicateLog := ctrl.LoggerFrom(ctx).WithValues("controller", "net-operator")
```

Comment:
The idea is for this k/v pair to match the logger that the reconciler gets, so I think this should be
```suggestion
	predicateLog := ctrl.LoggerFrom(ctx).WithValues("controller", "networkinterface")
```
(lower case kind of For, if Named doesn't overwrite it)

Reply:
If we don't match this exactly filtering on the controller name doesn't work (or we have to filter  on multiple controller names)